cmd/kubeadm/app/cmd/join.go

```diff
@@ -112,7 +112,7 @@ var (
 		certificate signing request (CSR) for a locally created key pair. By
 		default, kubeadm will set up the Kubernetes Control Plane to automatically
 		approve these signing requests. This token is passed in with the
-		--tls-bootstrap-token abcdef.1234567890abcdef flag.
+		--tls-bootstrap-token flag.
```

Comment:
i think this is fine as is. the description above also includes some token examples.
..and users should understand that these are just examples of tokens.

/close